doc/python/ternary-plots.md

```diff
@@ -33,7 +33,35 @@ jupyter:
     thumbnail: thumbnail/v4-migration.png
 ---
 
-### Basic Ternary Plot with Markers
+## Ternary Plots
```

Comment:
this part does not seem related to the rest of the PR but it seems ok...

Reply:
sorry, should have been targeted to `doc-prod`